zone/lua_general.cpp

```diff
@@ -361,6 +361,18 @@ bool lua_is_paused_timer(const char *timer) {
 	return quest_manager.ispausedtimer(timer);
 }
 
+bool lua_has_timer(const char *timer) {
+	return quest_manager.hastimer(timer);
+}
+
+uint32 lua_get_remaining_time(const char *timer) {
+	return quest_manager.getremainingtimeMS(timer);
+}
+
+uint32 lua_get_timer_duration(const char *timer) {
+	return quest_manager.gettimerdurationMS(timer);
+}
+
```

Comment:
Missing the luabind::def for all of these.

Reply:
ok think I added it correctly now.

Reply:
The ones you added are copy and paste of void methods. Have to set as the proper type and number of arguments.

Reply:
```cpp
luabind::def("has_timer", (bool(*)(const char*))&lua_has_timer),
luabind::def("get_remaining_time", (uint32(*)(const char*))&lua_get_remaining_time),
luabind::def("get_timer_duration", (uint32(*)(const char*))&lua_get_timer_duration),
```

Reply:
ok